.github/workflows/git-artifacts.yml

```diff
@@ -0,0 +1,313 @@
+name: git-artifacts
+
+on:
+  workflow_dispatch:
+    inputs:
+      build_only:
+        description: 'Optionally restrict what artifacts to build (portable, installer, etc.). Separate artifacts with spaces'
+        required: false
+      ref:
+        description: 'Optionally override which branch to build'
+        required: false
+        default: main
+      repository:
+        description: 'Optionally override from where to fetch the specified ref'
+        required: false
+        default: git-for-windows/git
+      architecture:
+        type: choice
+        description: 'Architecture to build'
+        required: true
+        options: 
+        - x86_64
+        - i686
```

Comment:
I chose to support only a single architecture at a time. Supporting multi-arch builds would make the workflow slightly more complex, but it's definitely doable. WDYT?

I'll add `arm64` in a follow-up PR because `please.sh build-mingw-w64-git` doesn't support that architecture yet. https://github.com/git-for-windows/build-extra/pull/452 will add that support.

Reply:
> Supporting multi-arch builds would make the workflow slightly more complex, but it's definitely doable.

I would love to have a multi-arch build, much [like we do for the MINGW-packages](https://github.com/git-for-windows/git-for-windows-automation/blob/e9b20e2697a348fd7c078fbe6a8f3c05e315a734/.github/workflows/build-and-deploy.yml#L254).

Reply:
On second thought, this presents a fine opportunity to clean things up and to use `git-sdk-32` to build the `mingw-w64-i686-git` packages...

Reply:
Yep, I think you're right, @dennisameling, we should support only a single architecture at a time. That will make things so much easier.